vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -132,6 +132,7 @@ quantitykind:AbsoluteTypographicMeasurement
   qudt:iec61360Code "0112/2///62720#UAD372" ;
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/quantitykind> ;
   rdfs:label "absolute typographic measurement" ;
+  skos:broader quantitykind:Distance ;
```

Comment:
This is the only occurrence of AbsoluteTypographicMeasurement in the repo, so it is not easy to decide by example, but I believe the correct broader qk would be qk:Length

Reply:
Agreed! For some reason I had read "Topographic" and concluded there must be a datum involved, as with Distance.
Now fixed.